generators/server/templates/src/main/java/package/config/_UaaConfiguration.java

```diff
@@ -87,9 +91,8 @@ public void configure(ClientDetailsServiceConfigurer clients) throws Exception {
          */
         clients.inMemory()
             .withClient("web_app")
-            .scopes("web-app")
+            .scopes("openid")
```

Comment:
@xetys sorry, I still can't see it. If there's something to fix then you're welcome to do it or send me a line number link and a description of what needs to change. I'm pretty sure the token request uses the client_id--not the scope.